tox.ini

```diff
@@ -31,6 +31,15 @@ deps =
     pytest
 commands = mypy src/ tests/
 
+[testenv:asan]
+setenv=
+    PYTHONDEVMODE=1
+    PYTHONMALLOC=malloc
+    CFLAGS=-lasan -fsanitize=address -fno-omit-frame-pointer
+allowlist_externals=bash
+commands=
+    bash -c 'export LD_PRELOAD=$(find /usr/lib -name "libasan.so*" | head -n 1) && printenv LD_PRELOAD && python -c "import dnaio" && pytest tests'
```

Comment:
This crashes for me because apparently the `find ... | head ...` appears to be fragile. I found [this SO answer](https://stackoverflow.com/a/47022141), which recommended using `gcc -print-file-name=libasan.so` instead. That worked for me.

Reply:
Fixed!